randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -400,61 +404,88 @@ Extra - room_id: [13]
   * Layers: default
   * L0 Hatch to Mimic Storage/Door to Frozen Tower
   * Extra - door_idx: (24,)
+  > Door to Data Room
+      All of the following:
+          Space Jump
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Hi-Jump and Damage Runs (Advanced) and Cold Damage ≥ 238
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 319
```

Comment:
what did you do to measure this, i cant replicate this at all. i'm taking 41 damage here by just falling down. are you facetanking all ripppers and blue x? the connection from the dat room to botom only takes ~150 so somethings definitely weird here